components/nds-tag/nds-tag.d.ts

```diff
@@ -13,7 +13,7 @@ declare module "@nice-digital/nds-tag" {
 		/** Use the .tag--outline modifier for a tag with a border. Use outline tags with filtered lists to show the currently applied filters. */
 		outline?: boolean;
 		/** Use removeable tags when users need to be able to remove or dismiss a tag. Use with outline tags at the top of filtered lists to show the currently applied selections. */
```

Comment:
remove/removeable?

Reply:
Fair but a minor point that we probably don't want to delay the release for